src/scss/_material-design.scss

```diff
@@ -0,0 +1,30 @@
+.md-underline {
```

Comment:
I create this file to group global styling that can be use in different custom components